tests/pytests/test_language.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+from common import getConnectionByEnv, waitForIndex
+from RLTest import Env
+
+def testSearchHashIndexLanguage(env):
+    conn = getConnectionByEnv(env)
+
+    # Create index - language Italian by default
+    env.cmd('FT.CREATE', 'idx', 'ON', 'HASH', 'PREFIX', '1', '{word}:',
+            'LANGUAGE', 'italian', 'SCHEMA', 'word', 'TEXT')
+    
+    # Create sample data in Italian
+    conn.execute_command('HSET', '{word}:1', 'word', 'arancia') # orange
+    conn.execute_command('HSET', '{word}:2', 'word', 'arance') # oranges
+
+    # Create sample data in English
+    conn.execute_command('HSET', '{word}:3', 'word', 'orange') # orange
+    conn.execute_command('HSET', '{word}:4', 'word', 'oranges') # oranges
+
+    # Wait for index to be created
+    waitForIndex(env, 'idx')
```

Comment:
`HSET` is synchronous, so no need for the wait

Reply:
Thanks. The `waitForIndex()` is now called after the `FT.CREATE`, in that case it is required to avoid a flaky test.